### PR TITLE
Bugfix: Fix wrong request calls.

### PR DESCRIPTION
**Pull request by cschra**
#Fix some leftover bugs of Laravel Upgrade and moving to the Request-Facade

### CODE REVIEW

File: modules/ProvBase/Http/Controllers/ConfigfileController.php [hunk 1]
```diff
@@ -194,22 +232,23 @@ public function recreateTree($content, $hasName, $originalConfigfiles)
      *
      * @author Roy Schneider
      * @param array $content
```
Comment (NinoRy) [inline modules/ProvBase/Http/Controllers/ConfigfileController.php]: the param for this comment has changed!

Comment (cschra) [reply modules/ProvBase/Http/Controllers/ConfigfileController.php]: fixed in ddea3c4

Comment (NinoRy) [reply modules/ProvBase/Http/Controllers/ConfigfileController.php]: still not fixed!

File: modules/ProvBase/Http/Controllers/ConfigfileController.php [hunk 2]
```diff
@@ -194,22 +235,23 @@ public function recreateTree($content, $hasName, $originalConfigfiles)
...
```
Comment (NinoRy) [inline modules/ProvBase/Http/Controllers/ConfigfileController.php]:
```suggestion
     * @param array $configfile
```

Comment (cschra) [reply modules/ProvBase/Http/Controllers/ConfigfileController.php]: fixed in ecb76a0

**Pull request merged**